src/main/java/me/moodcat/database/entities/Room.java

```diff
@@ -45,28 +51,53 @@
     private Integer id;
 
     /**
+     * The name of the room.
+     */
+    @Column(name = "roomName")
+    private String roomName;
```

Comment:
Then we should choose to use either roomName or name, but not both. Personally I think name is more logical, as we already know it is a name field in the room class.